VmAgent.Core/Interfaces/SessionHostContainerConfiguration.cs

```diff
@@ -46,7 +46,11 @@ protected override string GetLogFolder(string logFolderId, VmConfiguration vmCon
         {
             // The VM host folder corresponding to the logFolderId gets mounted under this path for each container.
             // So the logFolderId itself isn't of much significance within the container.
-            return vmConfiguration.VmDirectories.GameLogsRootFolderContainer + Path.DirectorySeparatorChar;
+
+            string vmDirGameLogPath = vmConfiguration.VmDirectories.GameLogsRootFolderContainer;
+            char pathSeperator = vmDirGameLogPath.StartsWith("/data/") ? Path.AltDirectorySeparatorChar : Path.DirectorySeparatorChar;
```

Comment:
is this change only for LocalMultiplayerAgent? code here will make it run on all customer VMs as well

Reply:
Thanks for the feedback Dimitris, I didn't know this is part of library currently used by customers. I will update it.